aviary/validation_cases/benchmark_tests/test_bench_multiengine.py

```diff
@@ -111,7 +120,7 @@ def test_multiengine_static(self):
         alloc_descent = prob.get_val('traj.descent.parameter_vals:throttle_allocations')
 
         assert_near_equal(alloc_climb[0], 0.5, tolerance=1e-2)
-        assert_near_equal(alloc_cruise[0], 0.56, tolerance=1e-2)
+        assert_near_equal(alloc_cruise[0], 0.64, tolerance=1e-2)
```

Comment:
why did this change?

Reply:
Engine sizing is working now! The engines are actually sized now, so instead of the initial guess of 0.5 scale factor for each engine, they get optimized and end up with one around ~0.35 and the other ~0.65